salt/master.py

```diff
@@ -224,9 +224,10 @@ def run(self):
 
         # Make Start Times
         last = int(time.time())
-        last_git_pillar_update = last
-
+        # update git_pillar on next loop
         git_pillar_update_interval = self.opts.get('git_pillar_update_interval', 0)
+        last_git_pillar_update = last - git_pillar_update_interval + self.loop_interval
```

Comment:
Alternatively, we can set `last_git_pillar_update = 0`, i.e. update `git_pillar` on first loop.

- Pros: code is simpler
- Cons: Higher load on start?